pilot/pkg/security/model/authentication.go

```diff
@@ -68,35 +68,6 @@ const (
 	AuthnFilterName = "istio_authn"
 )
 
-// MutualTLSMode is the mutule TLS mode specified by authentication policy.
-type MutualTLSMode int
```

Comment:
what's the motivation to move code around for these enums?

Reply:
I want to use this to store the (processed) mTLS mode to  `AuthenticationPolicies.namespaceMTLSMode` so it can be used directly with policy_applier later.